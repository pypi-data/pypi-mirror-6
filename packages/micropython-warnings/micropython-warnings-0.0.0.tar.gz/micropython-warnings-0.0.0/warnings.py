# MicroPython doesn't support warnings
