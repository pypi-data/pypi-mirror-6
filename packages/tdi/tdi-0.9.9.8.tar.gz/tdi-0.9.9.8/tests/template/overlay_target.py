#!/usr/bin/env python
import warnings as _warnings
_warnings.resetwarnings()
_warnings.filterwarnings('error')

from tdi import html

template = html.from_string("""
<node tdi="item">
    <znode tdi="nested" tdi:overlay="foo">
        <ynode tdi="subnested"></ynode>
    </znode>
    <xnode tdi="a"></xnode>
</node>
""".lstrip()).overlay(html.from_string("""
<anode tdi="grumpf" tdi:overlay="&gt;foo">
    <bnode tdi="gna"></bnode>
</anode>
""".lstrip())).overlay(html.from_string("""
<anode tdi="grumpf2" tdi:overlay="<foo">
    <cnode tdi="gna2"></cnode>
</anode>
""".lstrip())).overlay(html.from_string("""
<anode tdi="grumpf3" tdi:overlay="<foo">
    <dnode tdi="gna3"></dnode>
</anode>
""".lstrip()))

class Model(object):
    def render_nested(self, node):
        node['been2'] = u'here2'

    def render_gna2(self, node):
        node.content = u"whoa!2"

model = Model()
template.render(model)
