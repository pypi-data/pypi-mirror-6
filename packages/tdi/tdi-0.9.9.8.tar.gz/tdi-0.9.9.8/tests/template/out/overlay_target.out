<node>
    <anode been2="here2">
    <cnode>whoa!2</cnode>
</anode>
    <xnode></xnode>
</node>
